src/app_service/service/community/service.nim

```diff
@@ -903,7 +903,7 @@ QtObject:
       error "Error importing the community: ", msg = e.msg
       # We should apply some notification mechanism on the application level which will deal with errors and
       # notify user about them. Till then we're using this way.
-      self.events.emit(SIGNAL_COMMUNITY_IMPORTED, CommunityArgs(error: e.msg))
+      self.events.emit(SIGNAL_COMMUNITY_IMPORTED, CommunityArgs(error: "The community private key is invalid"))
```

Comment:
is this the only possible reason for error when importing a community?

Reply:
I could make the message more generic: "Error while importing the community!

BTW i kept the error message in the log
wdyt?

Reply:
I think thats better. 
Error message in log looks okay :)

Reply:
Message updated ;)